dask_ml/wrappers.py

```diff
@@ -551,3 +551,141 @@ def _predict_proba(part, estimator):
 
 def _transform(part, estimator):
     return estimator.transform(part)
+
+
+try:
+    from scikeras.wrappers import KerasRegressor, KerasClassifier
+except:
+
+    class _KerasError:
+        def __init__(self, *args, **kwargs):
+            raise ModuleNotFoundError(
```

Comment:
I'll need to check the intended usage, but what's the advantage to delaying the error until class creating time rather than import time? As a user I think it'd be best to know immediately, and I think the code is simpler.

The potential issue I see: what if I just want to use torch and not tensforflow? In that case we'd need to do something fancier.

Perhaps we require Python 3.7 and use https://www.python.org/dev/peps/pep-0562/.